.dadew

```diff
@@ -5,7 +5,7 @@
         "7zip",
         "bazel",
         "cacert",
-        "curl-7.65.3",
+        "curl-7.73.0",
```

Comment:
Maybe just drop the version number from the file name?

Reply:
I think we should either strip it everywhere or add it everywhere. The current mixture is a mess. @aherrmann-da Is there an advantage to adding the suffix? Does it make upgrades more robust in some way?

Reply:
I'm not aware of an advantage that would be relevant to us. Dropping the suffix seems fine to me.

Reply:
Great. Let's get this in and then we can delete all the version numbers later.